.gitignore

```diff
@@ -127,3 +127,7 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
```

Comment:
We probably don't want to include all these, given that this is a Pants repo. I recommend using the same file as `example-python`: https://github.com/pantsbuild/example-python/blob/main/.gitignore

(Ik this was added already with the first commit, not by you)

Reply:
Related, you're missing some Pants things like `/dist`

Reply:
Uses the example-python copy now.